tests/bundle/test_bundle.py

```diff
@@ -88,7 +88,7 @@ def test_max_allowed_ops_unstaked_sender(w3, helper_contract):
     wallet = deploy_wallet_contract(w3)
     calldata = wallet.encodeABI(fn_name="setState", args=[1])
     wallet_ops = [
-        UserOperation(sender=wallet.address, nonce=hex(i), callData=calldata)
+        UserOperation(sender=wallet.address, nonce=hex((i << 64)), callData=calldata)
```

Comment:
Is this the right approach? Sending ops with increasing nonces to the bundler under the same key will result in a failed simulation with error `AA25 invalid account nonce`.

Reply:
Yes that's an issue, we're working to solve it.